### PR TITLE
add 4.3 inch display mount for switchwire

### PR DESCRIPTION
**Pull request by Gi7mo**
* [X] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [X] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [X] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [X] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [X] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [X] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [X] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [X] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [ X] Submitted STL files are oriented and scaled properly for printing.
  * [X] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
added a switchwire 4.3" display mount in addition to jeoje already existing one for the 2.4
so credit still goes to him as i only added a mount for the switchwire and rotated the faceplate

#### How was it tested? 
printed and isntalled on my swirchwire

### CODE REVIEW

File: printer_mods/README.md [hunk 1]
```diff
@@ -125,7 +125,7 @@ like so:
 | | [V0 beltchains](./Iakabos/V0_beltchain)|Modified beltchain parts for V0 Z cables|:heavy_check_mark: |:x: |:x: |:x:|
 |jasonc |[V2.4 Z-Idler Shoulder Bolt Mod](./jasonc/Z_Idler_Shoulder_Bolt) |Modification to the idler carriers to accept a 25mm shoulder bolt. Ideal for Gates idlers. |:x: |:x: |:heavy_check_mark: |:x:|
 |jeoje |[V2.4 Z Chain Guide with Integrated Chamber Thermistor Mount](./jeoje/Z_Chain_Guide_Thermistor_Mount) | Modifies the Z chain guide to hold a chamber thermistor  |:x: |:x: |:heavy_check_mark: |:x:|
-| | [V2.4 4.3" Touchscreen Mount](./jeoje/4.3_Inch_Touchscreen_Mount) | Mounts a [Waveshare 4.3inch Capacitive Touch Display](https://www.waveshare.com/4.3inch-dsi-lcd.htm) to v2.4 Vorons |:x:|:x:|:heavy_check_mark:|:x:|
+| | [V2.4 4.3" Touchscreen Mount](./jeoje/4.3_Inch_Touchscreen_Mount) | Mounts a [Waveshare 4.3inch Capacitive Touch Display](https://www.waveshare.com/4.3inch-dsi-lcd.htm) to v2.4 Vorons |:x:|:x:|:heavy_check_mark:|:heavy_check_mark:|
```
Comment (KiloQubit) [inline printer_mods/README.md:128]: If this is now compatible with a Switchwire, you should modify the title and description here to remove V2.4.

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your mod!

Have you checked with @jeoje if it's ok to modify their mod?  I will need @jeoje to comment on this PR and confirm that this is ok before I can merge it.

The logo on the faceplate prints in the air:
Screen Shot 2021-08-07 at 09 28 37

Also see a README change below.

File: printer_mods/README.md (hunk 1 above)
Comment (jeoje) [reply printer_mods/README.md:128]: I don't have a problem with this. Thanks for the ping. Also, that part of the logo prints 1 layer off the bed intentionally.

Comment (Gi7mo) [reply printer_mods/README.md]: i've updated the readme
sorry for not pinging you @jeoje and thanks for the allowance

**Review by KiloQubit — APPROVED** [review]
Thanks for the updates!

**Pull request merged**